packages/catalog-process/open-api/catalog-service-spec.yml

```diff
@@ -200,7 +200,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/EService"
+                $ref: "#/components/schemas/ReturnId"
```

Comment:
minor, but I'm not sure we want to return id's for update requests (on E-Services or descriptors or documents): isn't it always the same id the caller used? then maybe an empty response is clearer